packages/KDropdownMenu/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@kongponents/kdropdownmenu",
+  "version": "0.1.0",
+  "description": "Dropdown Menu component",
+  "main": "dist/KDropdownMenu.umd.min.js",
+  "componentName": "KDropdownMenu",
+  "source": "KDropdownMenu.vue",
+  "files": [
+    "dist",
+    "*.vue",
+    "index.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Kong/kongponents.git"
+  },
+  "author": "Kong Inc",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/Kong/kongponents/issues"
+  },
+  "homepage": "https://github.com/Kong/kongponents#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
```

Comment:
If we leave the Vue-router links, I believe we'd need to add `vue-router` as a peer dependency (not ideal)

Reply:
Add a peer dependecy on `vue-router@^3.5.4` and update the other peer dependencies (`KRumbs/package.json` and the root `package.json`)

Reply:
Done!